### PR TITLE
[37] About Me- Fullstack Track #446

### PR DESCRIPTION
**Pull request by AAOYV2020**
## Task: `aboutMe`
- `I wrote my goals for taking on this program`

### CODE REVIEW

File: submissions/AnthoniaAdollo/aboutMe/AboutME.rtf [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{\rtf1\ansi\ansicpg1252\cocoartf2509
```
Comment (ToluAkin) [inline submissions/AnthoniaAdollo/aboutMe/AboutME.rtf:1]: @AAOYV2020 You need to delete this file from your pull request. It is not part of the success criteria.

File: submissions/AnthoniaAdollo/aboutme.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Hello World,
```
Comment (ToluAkin) [inline submissions/AnthoniaAdollo/aboutme.txt:1]: @AAOYV2020 The aboutme text file should be in the aboutMe folder you have created. Also, the name convention for your name and the name of the text file do not follow the required naming convention as in `camelCase`. Well done on coming this far.

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@AAOYV2020 You would need to close this pull request and create another one because the task title should be the only set of numbers that should be on the pull request title. For this task, the pull request title should be `[37] About Me Full Stack Track`

File: submissions/AnthoniaAdollo/aboutme.txt (hunk 2 above)
Comment (AAOYV2020) [reply submissions/AnthoniaAdollo/aboutme.txt:1]: Thank you for the feedback, Tolu. I have updated the naming convention on the 'Name' folder to camelCase. I apologise for the error. As for the text file, the one for the pull request was actually inside the 'aboutMe' file, as directed. the one outside was created while I was following folashade's tutorial. I will take that out, too

File: submissions/AnthoniaAdollo/aboutMe/AboutME.rtf (hunk 1 above)
Comment (AAOYV2020) [reply submissions/AnthoniaAdollo/aboutMe/AboutME.rtf:1]: I'm a tad confused. This is the 'aboutMe' folder I created within the name folder. I need some help with navigating this, please.

**Pull request closed without merging**